Python/ki/__init__.py

```diff
@@ -1178,7 +1178,13 @@ def OnScreenCaptureDone(self, image):
 
         PtDebugPrint(u"xKI.OnScreenCaptureDone(): Saving image to \"{}\".".format(tryName), level=kWarningLevel)
         if "saveAsPNG" in dir(image):
```

Comment:
Unrelated, but `if hasattr(image, "saveAsPNG"):` :wink:

Reply:
Done and gone! 🥇